.github/workflows/python-package.yml

```
@@ -18,20 +18,26 @@ jobs:
       matrix:
         os: [ ubuntu-latest, windows-latest, macos-latest, ubuntu-24.04-arm ]
         python-version: [ "3.10", "3.11", "3.12", "3.13", "3.14" ]
 
     steps:
       - uses: actions/checkout@v4
+      - name: Install uv
+        uses: astral-sh/setup-uv@v5
+        with:
+          enable-cache: true
       - name: Set up Python ${{ matrix.python-version }}
         uses: actions/setup-python@v4
```

actionlint (1.7.10)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)